ext_error.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/thinkgos/aliyun-iot/uri"
 )
 
+// @see https://help.aliyun.com/document_detail/120329.html?spm=5176.11065259.1996646101.searchclickresult.3703243801uSYu
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>line is 121 characters (lll)